src/imitation/scripts/config/train_preference_comparisons.py

```diff
@@ -19,11 +19,15 @@
 def train_defaults():
     fragment_length = 100  # timesteps per fragment used for comparisons
     total_timesteps = int(1e6)  # total number of environment timesteps
-    total_comparisons = 1000  # total number of comparisons to elicit
+    total_comparisons = 5000  # total number of comparisons to elicit
     # comparisons to gather before switching back to agent training
-    comparisons_per_iteration = 50
+    comparisons_per_iteration = 100
     # factor by which to oversample transitions before creating fragments
-    transition_oversampling = 10
+    transition_oversampling = 1
```

Comment:
OK with removing the warning

Reply:
I noticed I'd already built in the ability to suppress the warning with a parameter to `RandomFragmenter`, so I went with that instead of removing the code entirely. Assuming we implement human feedback at some point, we can then have it turned on by default when using that